langkit/templates/pkg_introspection_spec_ada.mako

```diff
@@ -68,21 +68,37 @@ package ${ada_lib_name}.Introspection is
    function Derived_Types (Id : Node_Type_Id) return Node_Type_Id_Array;
    --  Return type references for all direct derivations for Id
 
+   <% all_abstract = ctx.sorted_parse_fields + ctx.sorted_properties %>
+
+   type Abstract_Node_Data_Reference is
```

Comment:
While this is the Langkit name, maybe `Node_Data_Reference` is sufficient for the public API name, what do you think ?

Reply:
Good point, I’ll rename, thanks!